workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -200,6 +200,49 @@ def get_metric_all_runs(self, metric_type: str, variable: str) -> pd.Series:
         metric_name = self.metric_name(metric_type, variable)
         return self.metrics[self.metrics.metric == metric_name]
 
+    def get_metric_type_table(self, metric_type: str) -> pd.DataFrame:
```

Comment:
Rather than increasing the surface area of `RunMetrics`, I would move make this function into the "view" code (static_report.py). "nicely labeled" seems to imply this is related more to visualization than data loading. Adding more methods to the RunMetrics interface will make it harder to implement for new backends (e.g. a database or wandb).